sys/cheri/cheri.h

```diff
@@ -136,6 +136,14 @@ extern u_int	security_cheri_debugger_on_sandbox_unwind;
 extern u_int	security_cheri_sandboxed_signals;
 extern u_int	security_cheri_syscall_violations;
 extern u_int	security_cheri_bound_legacy_capabilities;
+
+#ifdef __CHERI_PURE_CAPABILITY__
+/*
+ * Used by the kernel linker to handle caprelocs in modules.
+ */
+void	init_linker_file_cap_relocs(void *start_relocs, void *stop_relocs,
+	    void *data_cap, void *pc_cap, unsigned long base_addr);
```

Comment:
Traditionally called `code_cap`, also `ptraddr_t` for `base_addr`, and should the start/stop pointers be `const`?

Reply:
Yes, those all seem sensible.  cheri/cheri_cap_relocs.c would have to be updated to match.